markdown/3.x/en/ff-checkout-tracking.md

```diff
@@ -10,9 +10,12 @@ The `count` attribute specifies how many items of the product were bought. It de
 
 The `price` attribute specifies the price of the bought item. If this attribute is not present, it will be inferred from the clicked item and the configured [fieldRoles](/documentation/3.x/field-roles).
 
+The `channel` attribute specifies the channel, tracking request should be send to. If this attribute is not present, default value configured in `ff-communication` will be used.
+
+
 ```HTML
 <ff-checkout-tracking>
-    <ff-checkout-tracking-item record-id="10321926" count="5"></ff-checkout-tracking-item>
+    <ff-checkout-tracking-item record-id="10321926" count="5" channel="NEW_CHANNEL"></ff-checkout-tracking-item>
```

Comment:
In the _Tracking Guide_ it is `channel="[NEW_CHANNEL]"`. For consistency I would suggest to remove `[ ]` in the _Tracking Guide_.

Reply:
changed